course05/homework02/freekpaans/.gitignore

```diff
@@ -0,0 +1,5 @@
+/build/
+/.clangd/
+/.projectile
+/.dir-locals.el
+/compile_commands.json
```

Comment:
Minor: as this is completely compiler / platform / etc dependent, it's generally not committed to the project iself (you can have a global gitignore file)

Reply:
That's what the `.gitignore` is for, right? Or are you saying you shouldn't have one at all? What else would be in there?